### PR TITLE
Feat(frontend):  add responsiveness

### PR DESCRIPTION
**Pull request by knodelle**
#5134 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by gbonaspetti — APPROVED** [review]
Approved for preview issue #4824

**Comment by gbonaspetti** [comment]
Tested on branch the preview issue #4824

File: frontend_lib/src/component/EmojiReactions/EmojiPickerButton.jsx
Comment (gbonaspetti) [inline frontend_lib/src/component/EmojiReactions/EmojiPickerButton.jsx:104]: Maybe do something like `autoFocus={!isMobile}`

**Review by gbonaspetti — APPROVED** [review]
Approved for #5166

**Comment by gbonaspetti** [comment]
Tested on branch the keyboard issue #5166

File: frontend_lib/src/component/Timeline/Timeline.jsx
Comment (knodelle) [reply frontend_lib/src/component/Timeline/Timeline.jsx:145]: nope

**Review by gbonaspetti — APPROVED** [review]
Approved for scroll issue #5169

**Comment by gbonaspetti** [comment]
Tested on branch the scroll issue #5169

**Review by gbonaspetti — APPROVED** [review]
Approved the image issue #5170

**Comment by gbonaspetti** [comment]
Tested on branch the image issue #5170

**Review by Haskaris — APPROVED** [review]
Approved the public name issue #5176

**Comment by Haskaris** [comment]
Tested on branch the public name issue #5176

File: frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx
Comment (gbonaspetti) [inline frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:121]:
```suggestion
                  id={rawTitle}
```

Comment (gbonaspetti) [reply frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:121]: Like that if we have two titles in the same page, we don't have the same id.

File: frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx
Comment (gbonaspetti) [inline frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:25]: Wording suggestion, because the component is not used only for Spaces, if you apply, please change all document.

Comment (gbonaspetti) [reply frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:25]:
```suggestion
      popoverTitleOpen: false
```

File: frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx
Comment (gbonaspetti) [inline frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:137]:
```suggestion
                  isOpen={state.popoverSpaceTitleOpen}
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx
Comment (knodelle) [reply frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:121]: Having the same id actually prevents having two popovers at the same time, because the last clicked rawTitle replaces the previous one, instead of showing two (I tried it out)

File: frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx
Comment (gbonaspetti) [inline frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:66]:
```suggestion
  handleTogglePopoverTitle = () => {
```

File: frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx
Comment (gbonaspetti) [inline frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:139]:
```suggestion
                  toggle={this.handleTogglePopoverTitle} // eslint-disable-line react/jsx-handler-names
```

File: frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx
Comment (gbonaspetti) [reply frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:121]: `id` is an unique identifier, so two HTML element should not have the same `id`. Or else the code is not following DOM standard (https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute) and it's (theoretically) not a valid HTML.

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx
Comment (gbonaspetti) [reply frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:121]: `id` is an unique identifier, so two HTML element should not have the same `id`. Or else the code is not following DOM standard (https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute) and it's (theoretically) not a valid HTML.

Comment (Haskaris) [reply frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx:121]: After speaking with @Skylsmoi 
The choice is to not merge this feature (add pop hover). I'd suggest to create another branch from commit `c0cacba` (I mean after this one, just before `b1573fc`) so we can safely merge previous modification without losing all our work and keep/use the second branch for R&D about a clean solution.

Since the `id` will be created from the title of a content/space, we'll require to have a slug function to remove bad characters, etc.

**Review by gbonaspetti — APPROVED** [review]

**Pull request merged**
